app/controllers/returns/NonExportedHumanMedicinesPlasticPackagingController.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.returns
+
+import connectors.CacheConnector
+import controllers.actions._
+import forms.returns.NonExportedHumanMedicinesPlasticPackagingFormProvider
+import models.Mode
+import models.requests.DataRequest
+import navigation.Navigator
+import pages.returns.{ExportedPlasticPackagingWeightPage, ImportedPlasticPackagingWeightPage, ManufacturedPlasticPackagingWeightPage, NonExportedHumanMedicinesPlasticPackagingPage}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.returns.NonExportedHumanMedicinesPlasticPackagingView
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class NonExportedHumanMedicinesPlasticPackagingController @Inject()(
+                                         override val messagesApi: MessagesApi,
+                                         cacheConnector: CacheConnector,
+                                         navigator: Navigator,
+                                         identify: IdentifierAction,
+                                         getData: DataRetrievalAction,
+                                         requireData: DataRequiredAction,
+                                         formProvider: NonExportedHumanMedicinesPlasticPackagingFormProvider,
+                                         val controllerComponents: MessagesControllerComponents,
+                                         view: NonExportedHumanMedicinesPlasticPackagingView
+                                 )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+
+  val form = formProvider()
+
+  private def nonExportedAmount(implicit request: DataRequest[_]) = {
+    val manufactured = request.userAnswers.get(ManufacturedPlasticPackagingWeightPage).getOrElse(0L)
```

Comment:
I wonder if in this case, where the question was not answered we should redirect the user to the home account page where it can start the journey again. In this situation probably something went wrong (because we cannot find the ManufacturedPlasticPackagingWeightPage,  ImportedPlasticPackagingWeightPage and ExportedPlasticPackagingWeightPagein the json). Just thinking at the case that happened in the past where user navigate to the page using URL (maybe they had the page cached). 

I think also that some of the pages work this way too.

Reply:
Ah thats good point, i suppose that should be a 'something went wrong' situation...